CHANGELOG.rst

```diff
@@ -51,6 +51,7 @@ Interactive improvements
 - Instead of flashing all the text to the left of the cursor, fish now flashes the matched token during history token search, the completed token during completion (:issue:`11050`), the autosuggestion when deleting it, and the full command line in all other cases.
 - Pasted commands are now stripped of any ``$`` prefix.
 - The :kbd:`alt-s` binding will now also use ``run0`` if available.
+- ``funced`` will now edit copied functions directly, instead of the file where it was copied. (:issue:`11614`)
```

Comment:
This is technically correct but maybe it's clearer to say
maybe « ... where "functions --copy" was invoked » ?
I'm fine eitehr way.